contracts/JoepegAuctionHouse.sol

```diff
@@ -90,7 +93,7 @@ contract JoepegAuctionHouse is
     /// - Number of seconds before an English Auction ends where any new
     ///   bid will extend the auction's end time
     /// - Number of seconds to extend an English Auction's end time by
-    uint256 public englishAuctionRefreshTime;
+    uint96 public englishAuctionRefreshTime;
```

Comment:
Keep it uint256 as it will use a full storage slot either way.

You can also change `englishAuctionMinBidIncrementPct ` to be both `uint128`. If so they will be packed together saving some gas when loading them together in a function

Reply:
I think its better to keep this as `uint96` b/c it helps me to avoid having to cast from `uint256` -> `uint96` in `_placeEnglishAuctionBid` on line 729 where I increment `auction.endTime` (uint96) by `englishAuctionRefreshTime`